static/js/components.ts

```diff
@@ -13,7 +15,21 @@ import * as keydown_util from "./keydown_util";
     }).get();
 */
 
-export function toggle(opts) {
+export function toggle(opts: {
+    html_class?: string;
+    values: {label: string; label_html?: string; key: string}[];
+    callback?: (label: string, value: string) => void;
+    child_wants_focus?: boolean;
+    selected?: number;
+}): {
+    maybe_go_left: () => boolean;
+    maybe_go_right: () => boolean;
```

Comment:
Why not use the same method syntax here (`maybe_go_left(): boolean`)?

Reply:
Fixed.